CONTRIBUTING.md

```diff
@@ -87,33 +87,33 @@ The success stories should be structured in the following way:
 
 You can read [Ahref's success story](https://ocaml.org/success-stories/peta-byte-scale-web-crawler) for an examplary success story.
 
-### <a name="content-user"></a> Add an Academic or Industrial User
+### <a name="content-user"></a><a name="academic-user"> Add an Academic or Industrial User</a>Industrial User<a name="industrial-user"></a>
 
 > Contribute to the [Academic Users](https://ocaml.org/academic-users) and [Industrial Users](https://ocaml.org/industrial-users).
 
 You can add a new academic user by creating a new Markdown file in [data/industrial_users/](data/industrial_users/). For instance: [cryptosense.md](data/industrial_users/en/cryptosense.md).
 
 You can add a new industrial user by creating a new Markdown file in [data/academic_institutions/](data/academic_institutions). For instance: [cornell.md](data/academic_institutions/en/cornell.md).
 
-### <a name="content-book"></a> Add a Book
+### <a name="content-book"></a><a name="books"> Add a Book</a>
 
 > Contribute to the [OCaml Books](https://ocaml.org/books).
 
 You can add a new OCaml book by creating a new Markdown file in [data/books/](data/books/). For instance: [ocaml-from-the-very-beginning.md](data/industrial_users/en/ocaml-from-the-very-beginning.md).
 
-### <a name="content-event"></a> Add an Event
+### <a name="content-event"></a><a name="meetup"> Add an Event</a>
 
 > Contribute to the [Community Events](https://ocaml.org/community).
 
 You can add a new community event by creating a new Markdown file in [data/meetups.yml](data/meetups.yml).
 
-### <a name="content-package"></a> Add a Featured Package
+### <a name="content-package"></a> <a name="package">Add a Featured Package</a>
 
 > Contribute to the [Featured Packages](https://ocaml.org/packages).
 
 To update the list of Featured Packages on the Packages page, you can update [data/packages.yml](data/packages.yml)
 
-### <a name="ocaml-changelog"></a> OCaml Changelog
+### <a name="ocaml-changelog"></a><a name="changleog"> OCaml Changelog</a>
```

Comment:
Notice how all of these headings already have an anchor that can be used. Seems like it is not necessary to add a new one.